homeassistant/components/auth/login_flow.py

```diff
@@ -208,8 +208,9 @@ async def _async_flow_result_to_response(
                     "invalid_code",
                 )
             ):
+                status_code = HTTPStatus.UNAUTHORIZED
                 await process_wrong_login(request)
```

Comment:
Is this still needed? the comment mentioned it was not working because we returned a 200 but now we return a 401.

Reply:
This is actually the most important line of this PR if I didn't misunderstood :)
Previously everything was just HTTPStatus.OK by default as this is the default value in `_prepare_result_json`, this overwrites this behaviour in the bad credential case.